src/admin/python/lsst/qserv/admin/replicationInterface.py

```diff
@@ -370,6 +369,25 @@ def ingest_data_file(
             )
         _log.debug("ingest file res: %s", res)
 
+    def build_table_stats(self, database: str, tables: List[str]) -> None:
+        for table in tables:
+            _log.debug("build table stats for %s.%s", database, table)
+            _post(
+                url=f"http://{self.repl_ctrl.hostname}:{self.repl_ctrl.port}/ingest/table-stats",
+                data=json.dumps(
+                    dict(
+                        database=database,
+                        table=table,
+                        row_counters_deploy_at_qserv=1,
+                        row_counters_state_update_policy="ENABLED",
+                        force_rescan=1,
+                        auth_key="",
+                        admin_auth_key="",
+                        instance_id="qserv_proj",
```

Comment:
Did you add this method `build_table_stats` for the integration test only? In theory, values of the last three properties (`auth_key`, `admin_auth_key`, and `instance_id`) should not be hard-coded because they depend on a specific setup (of Qserv and its Replication/Ingest system).

Reply:
right now this interface is only used by itest. It could in theory be more generally useful, though auth_key="" and admin_auth_key="" are used pretty generally in this file, though I could make them arguments to the itest entrypoint command. Where are the values for these set in the RI system?

qserv_proj is passed as an argument to the replication system executables via extended args to entrypoint commands in the docker-compose file, and so that should definitely be an argument to the itest entrypoint command as well. I'll do that.